procedures/README.md

```diff
@@ -0,0 +1,40 @@
+# Procedures Utility
+
+This repository contains the utility code for adding value sets to the snowstorm server.
```

Comment:
Doesn't this utility ALSO add data into Bahmni Server? If so, then the description needs to be updated.

Reply:
Updated